### DISCUSSION TITLE
Collapsible menu lost in v4

### DISCUSSION THREAD

**Discussion by tfboy** [netbox-community/netbox#17502] [Q&A]
In previous versions (v3.x) of Netbox, you could collapse the menu on the left and had the option of clicking on a pin to keep it expanded. When collapsed, there was only a thin vertical strip showing the icons that would then auto-expand on mouseover.

This seems to have gone with the new UI in version 4.

I did do a quick search but couldn't find any mention of this. Is this a new "restriction" that accompanies the new menu / redesign?

IMO, it's a shame as I'm often split-screening and need to have the maximum horizontal space available, and having a non-collapsible menu eats up valuable real-estate.

**Comment by jeremystretch** [comment]
Feature request #16392 is open for volunteers. Would you like to work on it?
